### PR TITLE
#3452 - Student landing page

### PR DESCRIPTION
**Pull request by katrina-aot-git**
Update the content on the student landing page.

Screenshot 2024-07-30 at 2 18 40 PM
Screenshot 2024-07-30 at 2 18 47 PM

### CODE REVIEW

File: sources/packages/forms/src/form-definitions/studentwelcomepage.json [hunk 1]
```diff
@@ -148,7 +148,7 @@
                       "value": ""
                     }
                   ],
-                  "content": "Your welcome page features helpful links which will allow you to start an application or manage your loan.",
+                  "content": "Welcome to your Student Account!\n<br><br>\nStudentAid BC helps eligible students with the cost of post-secondary education through student loans, grants, and other financial assistance programs. From this page you can:\n<ul>\n<li>Apply for student financial assistance.</li>\n<li>View your current and past applications and assessments</li>\n<li>Track your application status</li>\n<li>Upload and view documents</li>\n<li>Request a change to an active application</li>\n<li>Manage your StudentAid BC profile</li>\n<li>View your account activity</li>\n</ul>",
```
Comment (dheepak-aot) [inline sources/packages/forms/src/form-definitions/studentwelcomepage.json:151]: Please use the custom class `align-bullets` in the custom class section to ensure padding to the bullets.

image

image

**Comment by dheepak-aot** [comment]
Looks good. Just one minor comment.

File: sources/packages/forms/src/form-definitions/studentwelcomepage.json (hunk 1 above)
Comment (katrina-aot-git) [reply sources/packages/forms/src/form-definitions/studentwelcomepage.json:151]: updated:
image

**Review by dheepak-aot — APPROVED** [review]
Looks good. 👍

**Pull request merged**